### DISCUSSION TITLE
report --min-score=SCORE meaning of 'score' and how its calculated?

### DISCUSSION THREAD

**Discussion by munntjlx** [praetorian-inc/noseyparker#186] [Q&A]
This is really good news! I was wondering how the 'score' is calculated and how I can use it to suppress 'non interesting' things. Say below threshold x, but I have to understand threshold to use it!

**Comment by bradlarsen** [comment]
Currently, there is nothing in the open-source version of Nosey Parker that populates `score`.

In the internal version (which uses the same datastore format), it's populated by one of a couple different ML models, and is a value in the range $[0.0 .. 1.0]$, where a higher number indicates that a finding looks more like real secret.

We don't have immediate plans to release the denoiser models as open-source. We are looking to make output of these models available in [Chariot](https://www.praetorian.com/proactive-cybersecurity-technology/) both for web crawling and "regular" secret scanning of Git repositories later this summer.
